apps/dashr-brain-viewer/brainDATA.R

```diff
@@ -0,0 +1,59 @@
+pts <- read_mniobj("surf_reg_model_both.obj")[1]
+tri <- read_mniobj("surf_reg_model_both.obj")[2]
+intensities <- lapply(as.list(read.table('aal_atlas.txt')),as.list)[[1]]
+
+data=plotly_triangular_mesh(pts, tri, intensities,
+                            colorscale=DEFAULT_COLORSCALE, flatshading=FALSE,
+                            showscale=FALSE, reversescale=FALSE, plot_edges=FALSE)
```

Comment:
Just for legibility, it's good to keep lines under 80 characters, just do something like:

```
data <- plotly_triangular_mesh(
  pts, tri, intensities,
  colorscale=DEFAULT_COLORSCALE, flatshading=FALSE,
  showscale=FALSE, reversescale=FALSE, plot_edges=FALSE
)
```